packages/react/src/components/CTA/CardCTA.js

```diff
@@ -28,6 +28,14 @@ const CardCTA = ({
   // eslint-disable-next-line no-unused-vars
   const { style, ...cardProps } = otherProps;
 
+  const [thumbnailSize, setThumbnailSize] = useState(3);
+
+  const refImage = useCallback(node => {
+    if (node && node.parentElement) {
```

Comment:
```suggestion
    if (node?.parentElement) {
```

I think we can chain these together